src/Rules.UnitTests/LocalRuleMetadataProvider.RuleInfoDeserializerTests.cs

```diff
@@ -0,0 +1,80 @@
+  'Tags': [],
+  'DescriptionSections': [
```

Comment:
Previously none of the analyzers had any data in the `DescriptionSections`, so deserializing it wasn't a problem.
The newest JS/TS analyzer has data, and the deserialiation failed.